make/platform/Makefile.linux64

```diff
@@ -17,3 +17,5 @@
 
 # Makefile.linux64
 # Special AR and RANLIB definitions for XMT used to be here.
+
+CXXFLAGS += -frtti
```

Comment:
LLVM builds with -fno-rtti and you should too. Are you using isa<> ? You should be using the LLVM equivalent.